rewrite-graphql/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id("org.openrewrite.build.language-library")
+}
+
+val antlrGeneration by configurations.creating {
+    extendsFrom(configurations.implementation.get())
+}
+
+tasks.register<JavaExec>("generateAntlrSources") {
+    mainClass.set("org.antlr.v4.Tool")
+    
+    args = listOf(
+        "-o", "src/main/java/org/openrewrite/graphql/internal/grammar",
+        "-package", "org.openrewrite.graphql.internal.grammar",
+        "-visitor"
+    ) + fileTree("src/main/antlr").matching { include("**/*.g4") }.map { it.path }
+    
+    classpath = antlrGeneration
+}
+
+dependencies {
+    api(project(":rewrite-core"))
+    api("org.jetbrains:annotations:latest.release")
+    api("com.fasterxml.jackson.core:jackson-annotations")
+    
+    compileOnly(project(":rewrite-test"))
+    
+    implementation("org.antlr:antlr4-runtime:4.13.2")
```

Comment:
Depending on the order in which we merge, you might want to keep an eye on this PR as well:
- https://github.com/openrewrite/rewrite/pull/5910